src/Input/Silk.NET.Input.Common/Structs/TouchFinger.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Numerics;
+
+namespace Silk.NET.Input
+{
+    /// <summary>
+    /// Represents a touch finger.
+    /// </summary>
+    public readonly struct TouchFinger
+    {
+        /// <summary>
+        /// The index of the finger.
+        /// </summary>
+        public long Index { get; }
+
+        /// <summary>
+        /// The last known position of the finger.
+        /// </summary>
+        public Vector2 Position { get; }
+
+        /// <summary>
+        /// The last known normalized position of the finger.
+        /// </summary>
+        public Vector2 NormalizedPosition { get; }
+
+        /// <summary>
+        /// The last known speed of the finger.
+        /// </summary>
+        public Vector2 Speed { get; }
+
+        /// <summary>
+        /// The last known normalized speed of the finger.
+        /// </summary>
+        public Vector2 NormalizedSpeed { get; }
```

Comment:
Could there be some documentation in here about what normalized means in this specific context? In the case of graphics programming this could very well mean `-1 -> 1` or `0 -> 1`

Reply:
I added (0..1) and (-1..1). Is this enough? In general positions are normalized to range 0 to 1 and speed values to range -1 to 1 as it has a direction. I tried to add this information everywhere before but forgot it here I guess.